src/v2/view-builder/components/AuthenticatorEnrollOptions.js

```diff
@@ -42,7 +44,33 @@ const AuthenticatorRow = View.extend({
   },
   minimize: function() {
     this.$el.addClass('authenticator-row-min');
-  }
+  },
+  getTemplateData() {
+    let authenticatorUsageText;
+    const allowedFor = this.model.get('relatesTo').allowedFor;
+
+    if (allowedFor) {
+      switch (allowedFor) {
+      case 'any':
+        authenticatorUsageText = loc('oie.enroll.authenticator.usage.text.access.recovery', 'login');
+        break;
+      case 'recovery':
+        authenticatorUsageText = loc('oie.enroll.authenticator.usage.text.recovery', 'login');
+        break;
+      case 'sso':
+        authenticatorUsageText = loc('oie.enroll.authenticator.usage.text.access', 'login');
+        break;
+      }
+    }
+
+    return {
+      authenticatorUsageText,
+      iconClassName: this.model.get('iconClassName'),
```

Comment:
if these are coming from model then they should be auto available as `options`

Reply:
do you mean `this.options.model.get('iconClassName')`? I don't see the values directly under `options`. does it make a difference whether I get the value as `this.model.get.. or this.options.model.get`

Reply:
I meant you need not to create the object for existing values again. they should be available in the template to use as is.

For ex.. just extend the object instead.? 

```
    const data = View.prototype.getTemplateData.apply(this, arguments);
    
    //extend data with new attributes 
    return data
```